madprops/serializers.py

```diff
@@ -52,7 +54,12 @@ def to_representation(self, value):
         return result_dict
 
     def to_internal_value(self, data):
-        data_list = [{name: value} for (name, value) in data.items()]
+        try:
+            data_list = [{name: value} for (name, value) in data.items()]
+        except AttributeError:
+            raise ValidationError(
+                _('Properties should be a key value mapping'))
```

Comment:
Sorry, missed your reply previously.  I was under impression that these messages are intended solely for a debug and shouldn't be shown in UI directly.  (and thus we shouldn't encourage somebody to use them for some other purpose).  But, I could be wrong.